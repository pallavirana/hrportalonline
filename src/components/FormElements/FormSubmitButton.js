import React from 'react';
import withStyles from 'react-jss';
import styles from './FormElements.styles';

export const FormSubmitButton = ({classes, type, value}) => <>
    <button type={type} className={classes.button}>{value}</button>
</>;

export default withStyles(styles)(FormSubmitButton);