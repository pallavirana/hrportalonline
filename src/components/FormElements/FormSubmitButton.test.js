import React from 'react';
import { shallow } from 'enzyme';
import { FormSubmitButton } from "./FormSubmitButton";


describe('FormSubmitButton component', () => {
    it('shallow renders without crashing', () => {
      const wrapper = shallow(<FormSubmitButton classes={{}} />);
      expect(wrapper).toMatchSnapshot();
    });
  });