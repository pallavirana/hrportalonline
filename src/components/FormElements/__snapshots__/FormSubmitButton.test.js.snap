// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormSubmitButton component shallow renders without crashing 1`] = `ShallowWrapper {}`;
